.github/workflows/ci-full.yml

```diff
@@ -311,7 +310,7 @@ jobs:
       - name: Pack module
         run: make pack BRANCH=$TAG_OR_BRANCH SHOW=1
       - name: Configure AWS credentials
-        uses: aws-actions/configure-aws-credentials@v4
+        uses: aws-actions/configure-aws-credentials@v3
```

Comment:
May I ask why downgrading from `v4`?

Reply:
https://github.com/RedisBloom/RedisBloom/actions/runs/7870589399/job/21472095060#step:11:16

Some sort of compatibility issue due to nodejs requirements IIUC. 
v3 seems to work fine.